legal-api/src/legal_api/models/alternate_name.py

```diff
@@ -42,24 +50,37 @@ class NameType(BaseEnum):
             "start_date",
             "naics_key",
             "naics_code",
-            "naics_description"
+            "naics_description",
+            "business_start_date",
+            "dissolution_date",
+            "state",
+            "state_filing_id",
+            "admin_freeze",
+            "last_modified"
         ]
     }
 
     id = db.Column(db.Integer, primary_key=True)
-    identifier = db.Column("identifier", db.String(10), nullable=True)
+    identifier = db.Column("identifier", db.String(10), nullable=True, index=True)
     name_type = db.Column("name_type", db.Enum(NameType), nullable=False)
-    name = db.Column("name", db.String(1000), nullable=False)
+    name = db.Column("name", db.String(1000), nullable=False, index=True)
     bn15 = db.Column("bn15", db.String(20), nullable=True)
     start_date = db.Column("start_date", db.DateTime(timezone=True), nullable=False)
     end_date = db.Column("end_date", db.DateTime(timezone=True), nullable=True)
     naics_key = db.Column("naics_key", db.String(50), nullable=True)
     naics_code = db.Column("naics_code", db.String(10), nullable=True)
     naics_description = db.Column("naics_description", db.String(300), nullable=True)
+    business_start_date = db.Column("business_start_date", db.DateTime(timezone=True), default=datetime.utcnow)
+    dissolution_date = db.Column("dissolution_date", db.DateTime(timezone=True), default=None)
+    state = db.Column("state", db.Enum(State), default=State.ACTIVE.value)
+    admin_freeze = db.Column("admin_freeze", db.Boolean, unique=False, default=False)
+    last_modified = db.Column("last_modified", db.DateTime(timezone=True), default=datetime.utcnow)
+
 
     # parent keys
     legal_entity_id = db.Column("legal_entity_id", db.Integer, db.ForeignKey("legal_entities.id"))
     change_filing_id = db.Column("change_filing_id", db.Integer, db.ForeignKey("filings.id"), index=True)
+    state_filing_id = db.Column("state_filing_id", db.Integer, db.ForeignKey("filings.id"))
 
     # relationships
     legal_entity = db.relationship("LegalEntity", back_populates="_alternate_names")
```

Comment:
Can we add the relationships we've added here?

You should be able to reference legal_entity model.

filings relationship - https://github.com/bcgov/lear/blob/077126959a7e695724727ec6123ff81b7c29b697/legal-api/src/legal_api/models/legal_entity.py#L300-L305

documents relationship - https://github.com/bcgov/lear/blob/077126959a7e695724727ec6123ff81b7c29b697/legal-api/src/legal_api/models/legal_entity.py#L310